schemas/dll.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: dll
+  title: DLL
+  group: 2
+  short: These fields contain information about loaded process modules.
+  description: |-
+    These fields contain information about loaded process modules.
```

Comment:
I think it is, though.

Reply:
🤔 Ok, it was the first time I heard the expression "process module". Doing a quick search does indeed turn up some Microsoft documentation about DLLs loaded in a process.

But in Linux land, the "modules" terminology is used for kernel modules, which are out of scope here. 😂

So that's why I'm suggesting this alternate phrasing.

Reply:
If you see a problem with switching over to my proposed phrasing, then at least we should explicitly mention that this doesn't mean Linux Kernel module. Perhaps add this, below the bullet points:

> Note that these fields are not meant to capture details about Linux kernel modules. They're meant to capture details about dynamic libraries loaded by any process, including userland processes.